doc/manual/manual.tex

```diff
@@ -10501,6 +10501,91 @@ \subsubsection{Advection Stabilization Benchmarks}
+
+This benchmark tests the implementation of the anelastic shear wave velocity-to-temperature conversion derived by Yamauchi \& Takei~\cite{YT16} based on forced-oscillation experiments conducted at seismic frequencies on polycrystalline borneol. This anelasticity parameterization has been calibrated against a range of observational constraints on upper mantle temperature, attenuation and viscosity structure, using the surface wave tomography model of Priestley et al.\cite{P12} to constrain shear wave velocity (V$_S$) and the plate model of \cite{McK05} to estimate lithospheric temperature structure. The resulting V$_S$-to-temperature conversion accurately accounts for the strongly non-linear temperature dependence of V$_S$ at near-solidus conditions and is therefore especially useful for initializing models with accurate temperature structure in the upper $\sim$~400 km of the mantle. This benchmark is located in the folder \url{benchmarks/yamauchi_takei_2016_anelasticity}.
+
+\begin{figure}
+\begin{center}
+  \centering
+  \includegraphics[width=\textwidth]{cookbooks/benchmarks/anelasticity/YT16_benchmark.png}
+  \caption{\it $V_S$ as a function of temperature in the oceanic lithosphere. Dotted lines = digitized results from Fig. 20 of Yamauchi \& Takei~\cite{YT16}; solid lines = \aspect{} results; red = 50 km; blue = 75 km. Temperatures are taken from the plate model of McKenzie et al.~\cite{McK05} and $V_{S}$ from the surface wave tomography model of Priestley et al.\cite{P12}.}  
+  \label{fig:anelasticity}
+\end{center}
+\end{figure}
+
+The parameterization of Yamauchi \& Takei~\cite{YT16} defines $V_S$ as
+\begin{equation}
+V_S = \frac{1}{\sqrt{\rho J_1}} \left( \frac{1+\sqrt{1+(J_2/J_1)^2}}{2}\right)^{-\frac{1}{2}} \simeq \frac{1}{\sqrt{\rho J_1}}
+\label{eq:Vs}
+\end{equation}
+where $\rho$ is the density and $J_1$ and $J_2$ represent real and imaginary components of the complex compliance, $J^*$, which is a quantity describing the sinusoidal strain resulting from the application of a unit sinusoidal stress. $J_1$ represents the strain amplitude in phase with the driving stress, whilst the $J_2$ component is $\frac{\pi}{2}$ out of phase, resulting in dissipation. Density is calculated using the expression
+\begin{equation}
+\rho (P,T) = \rho_{0}  \left\{ 1 - \left[\alpha(T - T_0)\right] + \frac{P}{K} \right\}
+\label{eq:density_P}
+\end{equation}
+where $rho_{0} = 3291$~kg~m$^{-3}$ and $\alpha = 3.59 \times 10^{-5}$~K$^{-1}$ are the density and thermal expansivity corresponding to $T_{0} = 873$~K, $P$ = pressure and $K = 115.2$~GPa is the bulk modulus. $J_1$ and $J_2$ are expressed as
+\begin{equation}
+J_1(\tau_S^{\prime})= J_U \left[ 1 + \frac{A_B[\tau_S^{\prime}]^{\alpha_B}}{\alpha_B} + \frac{\sqrt{2\pi}}{2} A_P~\sigma_P \left\{ 1-\text{erf}\left(\frac{\text{ln}[\tau_P^{\prime}/\tau_S^{\prime}]}{\sqrt{2}\sigma_P}\right)\right\}\right]
+\label{eq:real_compliance}
+\end{equation}
+\begin{equation}
```

Comment:
I think it might be nicer to use the `{align}` environment and align the two equations for J1 and J2.

Reply:
Done